test/hotspot/jtreg/runtime/cds/appcds/sharedStrings/IncompatibleOptions.java

```diff
@@ -108,9 +108,12 @@ public static void test(String[] args_ignored) throws Exception {
             testDump(1, "-XX:+UseZGC", "-XX:-UseCompressedOops", null, false);
         }
 
-        // Dump heap objects with ParallelGC and SerialGC
+        // Dump heap objects with Parallel, Serial, Shenandoah GC
         testDump(2, "-XX:+UseParallelGC", "", "", false);
         testDump(3, "-XX:+UseSerialGC", "", "", false);
+        if (GC.Shenandoah.isSupported()) {
+            testDump(4, "-XX:+UseShenandoahGC", "", "", false);
+        }
```

Comment:
I didn't realize that Epsilon is also included in this PR. Should we add a test case for Epsilon as well?

Reply:
Sounds fair